qemu/tests/chardev_pci_serial_login.py

```diff
@@ -10,7 +10,7 @@
 @error_context.context_aware
 def run(test, params, env):
     """
-    Verify the login function of pci-serial (RHEL and x86 only):
+    Verify the login function of pci-serial (RHEL only):
```

Comment:
In fact, there has a weak issue --> `pci-serial`, I think maybe we can rename this file to `chardev_serial_login.py`.

Reply:
Hi yihuang,

This is the code developed in x86, and will be linked to the x86 cfg file and test loop. So I don't recommend modifying the py file name.

Reply:
You don't need to rename the test case, just only update the type in cfg.
https://github.com/autotest/tp-qemu/blob/080645d194c5df5816147bcde49204a0b8befb52/qemu/tests/cfg/chardev_pci_serial_login.cfg#L4

Reply:
Ok, Corrected.